storefront-reference-architecture-master/cartridges/app_storefront_base/cartridge/controllers/Default.js

```diff
@@ -22,26 +22,33 @@ var userLoggedIn = require('*/cartridge/scripts/middleware/userLoggedIn');
  * @param {renders} - isml
  * @param {serverfunction} - get
  */
-server.get('Start',userLoggedIn.validateLoggedIn, consentTracking.consent, cache.applyDefaultCache, function (req, res, next) {
-    var Site = require('dw/system/Site');
-    var PageMgr = require('dw/experience/PageMgr');
-    var pageMetaHelper = require('*/cartridge/scripts/helpers/pageMetaHelper');
-
-    pageMetaHelper.setPageMetaTags(req.pageMetaData, Site.current);
-
-    var page = PageMgr.getPage('homepage');
-
-    if (page && page.isVisible()) {
-        res.page('homepage');
-    } else {
-        res.render('home/homePage');
-    }
-    next();
-}, pageMetaData.computedPageMetaData);
+server.get(
+    "Start",
```

Comment:
You can revert all the app_storefront-base changes, as we agreed, this needs to stay untouched